spec/exporters/hathi/content_package_spec.rb

```diff
@@ -104,6 +104,17 @@
       expect(page.derivative_path.ftype).to eq "file"
     end
 
+    it "has a copy of the original image" do
+      image = package.pages.first.original_image
+      expect(image.mime_type).to eq "image/tiff"
+    end
+
+    # fix this: figure out how to get FactoryBot to create derivatives
```

Comment:
I believe you can use the `run_real_derivatives` directive like https://github.com/pulibrary/figgy/blob/master/spec/adapters/bagit/bag_exporter_spec.rb#L19

Reply:
Not sure what it's doing; the mime_type is still image/tiff, but I'd expect it to be image/jp2